pkg/blob/blob.go

```diff
@@ -1,12 +1,9 @@
 /*
 Copyright 2019 The Kubernetes Authors.
-
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
-
```

Comment:
revert the changes in this file

Reply:
Are these created by lint?

Reply:
yeah i'm not sure why those ended up there... will revert momentarily

Reply:
double checked my code, those dashes aren't there, might have been created by lint fixes as you said